straxen/plugins/led_calibration.py

```diff
@@ -11,20 +11,20 @@
 # are made available under straxen.[...]
 export, __all__ = strax.exporter()
 
-
+channel_list = [i for i in range(494)]
 @export
 @strax.takes_config(
     strax.Option('baseline_window',
                  default=(0,50),
                  help="Window (samples) for baseline calculation."),
     strax.Option('led_window',
-                 default=(75, 110),
+                 default=(75, 105),
                  help="Window (samples) where we expect the signal in LED calibration"),
     strax.Option('noise_window',
                  default=(20, 55),
                  help="Window (samples) to analysis the noise"),
     strax.Option('channel_list',
-                 default=(0,494),
+                 default=(channel_list),
```

Comment:
Hi Giovanni, maybe a naive question but wouldn't it be faster to add another plugin on top of this one to extract this information? Guessing from your description it looks like you'd want to reprocess this plugin using various versions of the channel_list. My guess would be that having this plugin doing all the channels as a default and extracting the required channels later would be much faster as you don't have to go back to raw_records each time?

Reply:
Thank you for the comment Joran, all the suggestions are very welcomed! As you pointed out it should be already kind of implemented. For every raw records chunk, which are quite fast to load, I am selecting only those PMTs belong to the **channel_list**.

Reply:
Hi, this would be a good idea for the stable calibration during the science data taking, once we established which channels are associated with the relative light level. I think something to associate the n-datasets of a single calibration to a defined list of pmts each, based on the filename of the dataset.